packages/admin/src/cli.ts

```diff
@@ -20,27 +20,32 @@ const COMMON_COMMAND_ARGUMENTS = {
     mnemonic: {
       type: 'string',
       demandOption: true,
-      describe: 'Mnemonic phrase for the Airnode master wallet',
+      describe: 'Mnemonic phrase for the Airnode wallet',
     },
     derivationPath: {
       type: 'string',
       describe: 'Derivation path to be used for deriving the wallet account',
     },
   },
-  requester: {
+  sponsor: {
     type: 'string',
     demandOption: true,
-    describe: 'Requester address',
+    describe: 'Sponsor address',
   },
-  airnodeId: {
+  airnode: {
     type: 'string',
     demandOption: true,
-    describe: 'The onchain ID of the Airnode instance',
+    describe: 'Address of the Airnode wallet',
```

Comment:
Is it airnode wallet? Isn't it an address where the airnode is deployed?

Reply:
It seems this is not a mistake... Can you explain how did we change using airnodeId and are using airnode wallet instead?

(Also, documentation in `WithdrawalUtils` for the airnode arguments could say `Airnode wallet address` instead of `Airnode address` to avoid confusion?)

Reply:
Previously we hashed the airnode master wallet address in order to get the airnodeId. Now that we no longer have an airnodeAdmin and master wallet is the default admin/owner of the airnode then there's really no need for an ID and we've decided to just use the wallet address as the "airnode id".
More on this here: https://github.com/api3dao/airnode/pull/433

Regarding WithdrawalUtils I can agree on adding the wallet word but keep in mind that this contract is being extended by airnode contract and it is not meant to be deployed or used on it's own.

Reply:
I see, makes sense now.